### PR TITLE
Improved design for the projects page

### LINKED ISSUE #346: Add student's project and improve the project page
**Actual Behaviour**
There is only 4 projects.

**Expected Behaviour**
Add another project and improve the design.

**Steps to reproduce it**

Add a student's project and improve the design.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sahilsaha7773**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.

# Changes done in this Pull Request

- [Preview Link](https://sahilsaha7773.github.io/projects.html)

- Fixes #346

## Description / Changes
I have improved the design of the students project page.

### CODE REVIEW

File: students_projects.html [hunk 1]
```diff
@@ -1,7 +1,17 @@
 ---
 layout: default-pages
 ---
+<style>
+    #back {
```
Comment (sk9331657) [inline students_projects.html]: Maks a separate project.css and add this to that file.

**Comment by sahilsaha7773** [comment]
ok, I will do the changes

**Comment by fragm3** [comment]
@sahilsaha7773 please make the changes so that this can be merged.

**Comment by sahilsaha7773** [comment]
Ok

**Pull request closed without merging**